### DISCUSSION TITLE
os.join_path('', '/usr/bin/x') returns 'usr/bin/x'

### DISCUSSION THREAD

**Discussion by mike-ward** [vlang/v#21748] [Questions and Answers]
`os.join_path('', '/usr/bin/x')` returns `usr/bin/x`. Is this expected?

**Comment by mike-ward** [comment]
Sorry, did I do something wrong? I wanted to check before reporting an issue.

**Reply by JalonSolov** [reply]
No, you didn't.  Looks like a bug to me.  I did the thumbs-down for the "Is this expected?" part.

**Comment by mike-ward** [comment]
Ah, OK. I wasn't trying to be snarky with the, 'is expected part'. I really was wondering if perhaps I misunderstood the intent of the function. I'll write up and issue. Again, sorry if I offended any one. Not my intent.

**Reply by JalonSolov** [reply]
I'm not communicating well, I guess.  No, I didn't take it as snarky.  I was just using a quick method of saying I didn't think it was expected.  No more, no less.
